charts/pulsar/templates/zookeeper/zookeeper-backup-statefulset.yaml

```diff
@@ -114,12 +114,12 @@ spec:
         {{- end }}
       terminationGracePeriodSeconds: {{ .Values.zookeeper.gracePeriod }}
       initContainers:
-      {{- with .Values.common.extraInitContainers }}
-      {{ toYaml . | indent 6 }}
-      {{- end }}
-      {{- with .Values.zookeeper.extraInitContainers }}
-      {{ toYaml . | indent 6 }}
-      {{- end }}
+{{- with .Values.common.extraInitContainers }}
+{{ toYaml . | indent 6 }}
+{{- end }}
+{{- with .Values.zookeeper.extraInitContainers }}
+{{ toYaml . | indent 6 }}
+{{- end }}
```

Comment:
Recommend using one-line style:

```
initContainers:
  {{- toYaml .Values.common.extraInitContainers | nindent 8 }}
  {{- toYaml .Values.zookeeper.extraInitContainers | nindent 8 }}
```

Reply:
`indent` is enough for this, right? We don't need a new line

Reply:
@zymap It won't add a new line here since we used the `{{-` to chomp whitespaces. 
The main purpose of using `nindent` is try best to keep the original YAML indentation for better code reading.

i.e., compare below two styles

- using `nindent`, the YAML file is:
  ```
  spec:
    initContainers:
      {{- toYaml .Values.common.extraInitContainers | nindent 8 }}
      {{- toYaml .Values.zookeeper.extraInitContainers | nindent 8}}
    containers:
      - name: ""
  ```

- using `indent`, the YAML file will be:
  ```
  spec:
    initContainers:
  {{ toYaml .Values.common.extraInitContainers | indent 8 }}
  {{ toYaml .Values.zookeeper.extraInitContainers | indent 8}}
    containers:
      - name: ""
  ```